be/src/olap/memtable.h

```diff
@@ -48,7 +49,8 @@ class MemTable {
 
     int64_t tablet_id() const { return _tablet->tablet_id(); }
     size_t memory_usage() const {
-        return _insert_mem_tracker->consumption() + _flush_mem_tracker->consumption();
+        return _insert_mem_tracker->consumption() + _arena->size() +
```

Comment:
If arena allocated some unused memory, then the memory usage maybe larger than actual usage. It will flush out many small segments.

Reply:
same as above